man/man5/zfs-module-parameters.5

```diff
@@ -652,7 +667,8 @@ Default value: \fB4M\fR.
 \fBzfs_dbuf_state_index\fR (int)
 .ad
 .RS 12n
-Calculate arc header index
+This feature is currently unused. It is normally used for controlling what
+reporting is available under /proc/spl/kstat/zfs.
```

Comment:
Good catch, this module option and the `state_index` parameter from `arc_buf_info()` should just be removed.  They're dead code.